akka-http-core/src/main/scala/akka/http/impl/engine/parsing/BodyPartParser.scala

```diff
@@ -289,5 +292,6 @@ private[http] object BodyPartParser {
     maxHeaderCount = 64,
```

Comment:
I noticed a TODO above this line `// TODO: load from config` - are we blocked on resolving it (i.e. no actor system here it seems) by something that needs discussion or just wasn't done yet?